src/omp/backend.f90

```diff
@@ -418,7 +418,9 @@ subroutine vecadd_omp(self, a, x, b, y)
       error stop "Called vector add with incompatible fields"
     end if
 
-    dims = size(x%data)
+    !dims = size(x%data)
+    ! Fix for size being stored wrongly into dims
+    dims = self%mesh%get_padded_dims(x)
```

Comment:
Remove comment and/or replace with explanation (to prevent people reverting back if non-obvious)

Reply:
done